.github/workflows/components-contrib-all.yml

```diff
@@ -0,0 +1,136 @@
+# ------------------------------------------------------------
+# Copyright 2021 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+name: "Build, Lint, Unit Test - complete matrix"
```

Comment:
Reviewers, this the old existing script - with the addition of the dispatch mechanism and schedule. This full version runs on release branches also.

Reply:
Similar to certification and conformance tests,  a PR comment be created if triggered by `ok-to-test`.

Reply:
Correct - that's how this work. But it also still runs this on release branches and tags.